src/matrix/DeviceMessageHandler.js

```diff
@@ -101,6 +105,19 @@ export class DeviceMessageHandler {
             }
         }
     }
+
+    _emitUnencryptedEvents(toDeviceEvents) {
+        const unencryptedEvents = toDeviceEvents.filter(e => e.type !== "m.room.encrypted");
+        for (const event of unencryptedEvents) {
+            this.emit("message", { unencrypted: event });
```

Comment:
```suggestion
            this.emit("message", event);
```
why not just work with an optional third `DecryptionResult` argument for encrypted events? Does a bit less allocations and makes it easier to listen for messages if you don't care whether they were encrypted or not.

So below you'd have:
```
            this.emit("message", result.event, result);
```